alpaca_trade_api/rest.py

```diff
@@ -69,11 +70,62 @@ def response(self):
             return self._http_error.response
 
 
-class TimeFrame(str):
-    Day = "1Day"
-    Hour = "1Hour"
-    Minute = "1Min"
-    Sec = "1Sec"
+class TimeFrameUnit(Enum):
+    Second = "Sec"
```

Comment:
The alpaca API actually doesn't support second bars.

Reply:
Thanks, I'll remove it. I based myself on the old `TimeFrame` enum and that sent me in the wrong direction. I'll send a fix soon.

Reply:
Changes applied in 9cb1b4d.